src/test/java/com/googlecode/lanterna/gui2/ComboBoxTest.java

```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) throws IOException, InterruptedException
 
     @Override
     public void init(WindowBasedTextGUI textGUI) {
-        final BasicWindow window = new BasicWindow("ComboBoxTest");
+        final BasicWindow window = new BasicWindow();
```

Comment:
Anyway, thanks a lot for your valuable contribution to the documentation!

On Sun, Nov 15, 2015 at 7:30 PM, Benjamin Dale <notifications@github.com>
wrote:

> In src/test/java/com/googlecode/lanterna/gui2/ComboBoxTest.java
> <https://github.com/mabe02/lanterna/pull/185#discussion_r44875532>:
>
> > @@ -15,7 +15,7 @@ public static void main(String[] args) throws IOException, InterruptedException
> >
> >      @Override
> >      public void init(WindowBasedTextGUI textGUI) {
> > -        final BasicWindow window = new BasicWindow("ComboBoxTest");
> > +        final BasicWindow window = new BasicWindow();
>
> Woops! Those shouldn't have gone in! My bad.
>
> I'll bring up a PR now which reverts their changes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mabe02/lanterna/pull/185/files#r44875532>.
>